examples/multiple_viewer_widget.py

```diff
@@ -2,21 +2,20 @@
 Multiple viewer widget
 ======================
 
-This is an example on how to have more than one viewer in the same napari window.
+This is an example of how to have more than one viewer in the same napari window.
 Additional viewers state will be synchronized with the main viewer.
 Switching to 3D display will only impact the main viewer.
 
-This example also contain option to enable cross that will be moved to the
+This example also contains the option to enable cross that will be moved to the
 current dims point (`viewer.dims.point`).
 
-.. tags:: gui
+.. Tags:: gui
```

Comment:
Is this change intentional/correct? Smells like indiscriminate autocorrect to me. 😂 

```suggestion
.. tags:: gui
```